batch/test/test_batch.py

```diff
@@ -1291,3 +1291,16 @@ def test_submit_update_to_deleted_batch(client: BatchClient):
         assert err.status == 404
     else:
         assert False
+
+
+def test_batch_submitted_by_second_user():
+    dev_client = BatchClient('test', token_file=os.environ['HAIL_TEST_DEV_TOKEN_FILE'])
+
+    try:
+        bb = dev_client.create_batch()
+        j = bb.create_job(DOCKER_ROOT_IMAGE, ['echo', 'test'])
+        b = bb.submit()
+        status = b.wait()
+        assert status['state'] == 'success', str((status, b.debug_info()))
+    finally:
+        dev_client.close()
```

Comment:
Should this test have another batch that is running at the same time by a different user as well?